docs/ecctl-release-notes.asciidoc

```diff
@@ -0,0 +1,50 @@
+[id="{p}-release-notes"]
+== Release notes
+
+This section summarizes the changes in each {n} release.
+
+* <<{p}-release-notes-1.0.0-beta1>>
+
+[id="{p}-release-notes-1.0.0-beta1"]
+=== Elastic Cloud Control (ecctl) 1.0.0-beta1
```

Comment:
```suggestion
=== Elastic Cloud Control (ecctl) v1.0.0-beta1
```

Reply:
@karencfv I've updated the release notes globally as per your comment. I see that `v1.0.0-beta1` is the tag you are using for this release, but it's somewhat different from the convention that other products use at Elastic. ECE was `1.0.0-beta1`, for example, no `v` prefix.

Reply:
ah! thanks for that information. I guess this is something that we might need to change in the future.